introspection-engine/introspection-engine-tests/src/test_api.rs

```diff
@@ -314,6 +314,15 @@ impl TestApi {
         expectation.assert_eq(&serde_json::to_string_pretty(&introspection_result.warnings).unwrap());
     }
 
+    #[track_caller]
+    pub async fn expect_no_warnings(&self) {
+        let previous_schema = psl::validate(self.pure_config().into());
+        let introspection_result = self.test_introspect_internal(previous_schema, true).await.unwrap();
+
+        dbg!(&introspection_result.warnings);
```

Comment:
This needs to be removed, right?

Reply:
Nope. It's in the test api, so if the assert panics underneath, this is printed out. It's a quite common way of giving debug output on assert falures.